imports/plugins/core/checkout/server/methods/workflow.js

```diff
@@ -13,22 +14,20 @@ import Reaction from "/imports/plugins/core/core/server/Reaction";
  * @param {String} userId - currently logged in user
  * @param {Object} selector - selector for product to update
  * @param {Object} modifier - Object describing what parts of the document to update.
- * @param {Object} validation
  * @return {String} _id of updated document
  * @private
  */
-function updateOrderWorkflow(userId, selector, modifier, validation) {
+function updateOrderWorkflow(userId, selector, modifier) {
   const order = Orders.findOne(selector);
 
   Hooks.Events.run("beforeUpdateOrderWorkflow", order, {
     userId,
-    modifier,
-    validation
+    modifier
   });
 
   Logger.debug("beforeUpdateOrderWorkflow hook executed before Order is updated");
 
-  const result = Orders.update(selector, modifier, validation);
```

Comment:
I'm guessing it would have been for passing options, and probably specifically the `{ bypassCollection2: true }` option since someone named it `validation`. Doesn't appear to be needed anymore, though.